src/libvfcfuncinstrument/libVFCFuncInstrument.cpp

```diff
@@ -0,0 +1,416 @@
+/******************************************************************************
+ *                                                                            *
+ *  This file is part of Verificarlo.                                         *
+ *                                                                            *
+ *  Copyright (c) 2015                                                        *
```

Comment:
Update copyright. Remove all previous and put simply:

Copyright (c) 2020
    Verificarlo contributors

Reply:
Fixed